source/agora/test/Base.d

```diff
@@ -1911,22 +1911,41 @@ public enum NetworkTopology
 public struct TestConf
 {
     /// Network topology to use
-    NetworkTopology topology = NetworkTopology.FullyConnected;
+    public NetworkTopology topology = NetworkTopology.FullyConnected;
 
     /// Extra blocks to generate in addition to the genesis block
-    size_t extra_blocks = 0;
+    public size_t extra_blocks = 0;
 
     /// Number of full nodes to instantiate
-    size_t full_nodes = 0;
+    public size_t full_nodes = 0;
 
     /// Number of extra validators which are initially outside the network
-    size_t outsider_validators = 0;
+    public size_t outsider_validators = 0;
 
     /// whether to set up the peers in the config
-    bool configure_network = true;
+    public bool configure_network = true;
 
-    /// Base values to use for the node configuration
-    NodeConfig node = {
+    /***************************************************************************
+
+        Base configuration for the nodes, matches `agora.common.Config : Config`
+
+    ***************************************************************************/
+
+    /// If the enrollments will be renewed or not at the end of the cycle
+    /// Matches the eponymous field in the `consensus` section.
```

Comment:
I had to look up `eponymous` but see it means 'named' in this context.

Reply:
Precisely, "with the same name", e.g. https://en.wikipedia.org/wiki/Metallica#:~:text=Its%20eponymous%20fifth%20album%2C%20Metallica%20(1991)